src/modules/Builder/features/ActivityItems/ItemConfiguration/Alerts/Alerts.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTranslation } from 'react-i18next';
+
+import { Svg } from 'shared/components';
+import { StyledBuilderBtn, theme } from 'shared/styles';
+
+import { Alert, items, options } from './Alert';
+import { AlertProps } from './Alerts.types';
+
+export const Alerts = ({ appendAlert, removeAlert, alerts }: AlertProps) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <>
+      {alerts.map((el, i) => (
+        <Alert key={el.id} {...el} index={i} removeAlert={removeAlert} />
+      ))}
+      <StyledBuilderBtn
```

Comment:
you can use Button variant="outlined"
for example  
![image](https://user-images.githubusercontent.com/116339462/224998941-75c535f9-8dd8-41e1-8dd0-148bc1e84bcd.png)
I think we don't need StyledBuilderBtn because it duplicates styles